packages/web/src/styles.css

```diff
@@ -3,16 +3,27 @@
 @tailwind utilities;
 
 @layer base {
+  /*
+    Both Hebrew and Greek fonts have Latin characters that we want to use other fonts to render.
+    Restricting the unicode range allows other fonts to render those characters instead.
+
+    Additionally, Hebrew and to a lesser extent Greek, are harder to read at standard font sizes,
+    so we scale them up a bit so that we don't have to set different font sizes for each language with css.
+   */
   @font-face {
     font-family: 'SBL-Hebrew';
     src: url('./assets/fonts/SBL_Hbrw.ttf') format('truetype');
     font-weight: normal;
+    unicode-range: U+0590-05FF;
+    size-adjust: 140%;
```

Comment:
I put a comment in the file about these to properties, but it is worth calling out here. Presently we are scaling up Hebrew two sizes from the base font and Greek one size. This enhances readability because the glyphs are smaller in the Hebrew and Greek fonts. By adjusting the size here in the font face, we no longer have to do that. We can use the same font sizes as English and trust that the size for Greek and Hebrew is equivalent for readability.

Additionally, by restricting the unicode range of each font to the characters specific to Greek and Hebrew, we can combine the SBL fonts with Noto Sans so that all characters are rendered by the right font. This means we can use the css class `font-mixed` to render content in all three languages. That's great for lexicon entries.

Reply:
Awesome! 


Where is `font-mixed` defined?